### PR TITLE
Changed example implementation and documentation for Combination graph to be carbon centric.

### LINKED ISSUE #167: Update documentation for Combination component to be carbon-centric
# Story

## Description

As a current consumer of carbon, I'd like for the public facing documentation for the Combination component to utilize carbon until terra-graphs is released

## Motivation

So that I will have the information I need to use the Combination component from carbon as things currently are released

## Definition of Done

We'll know we're done when the documentation in terra-graphs-docs is utilizing carbon instead of terra-graphs.

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Made changes to example implementation and documentation for all line graph examples to make the dev site carbon centric.

Closes #167 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

### Testing

- All existing tests are successful.

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination/BasicCombinationGraph/LineWithMultipleBubbleData.5.doc.mdx [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
 
 ## Usage
-```jsx
-import React from 'react';
-import CombinationGraph from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph';
+```js
+import Carbon from "@cerner/carbon-graphs/dist/js/carbon-graphs.js";
+import "@cerner/carbon-graphs/dist/css/carbon-graphs.css";
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination/BasicCombinationGraph/LineWithMultipleBubbleData.5.doc.mdx:17]: This and all the other places.

```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
```

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination/BasicCombinationGraph/LineWithMultipleBubbleData.5.doc.mdx]: Fixed in [this](https://github.com/cerner/terra-graphs/pull/181/commits/9eb4a3d4b7f59e64d194815b5394d3e8d8d95ba8) commit.

File: packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/CombinationGraph/Regions/Simple.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      start: 120,
+      end: 180,
+      color: '#c8cacb',
+    },
+  ],
+  low: [
+    {
+      axis: 'y2',
+      start: 20,
+      end: 80,
```
Comment (benbcai) [inline packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/CombinationGraph/Regions/Simple.jsx:37]: Does low also need a color like high?

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/CombinationGraph/Regions/Simple.jsx:37]: It is optional ben if the consumer does not provide any color then our default color(black) will come into the picture.

**Review by benbcai — APPROVED** [review]

File: packages/terra-graphs-docs/CHANGELOG.md [hunk 3]
```diff
@@ -20,3 +20,4 @@
 
 * Changed 
   * Moved core documentation to terra-dev-site.
+  * Changed example implementation and documentation for Combination graph to be carbon centric.
```
Comment (sdadn) [inline packages/terra-graphs-docs/CHANGELOG.md:23]: nit: `Updated examples and documentation for Combination graphs to be carbon centric.`

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/CHANGELOG.md]: Fixed [here](https://github.com/cerner/terra-graphs/pull/181/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R23).

**Review by tbiethman — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**